config/config.go

```diff
@@ -25,6 +25,9 @@ type Driver struct {
 	Token        string      `json:"token,omitempty"`
 	URL          string      `json:"url,omitempty"`
 	Protocol     string      `json:"protocol,omitempty"`
+
+	DisablePing    bool `json:"disable_ping,omitempty"`
+	PingIntervalMs int  `json:"ping_interval_ms,omitempty"`
```

Comment:
Can we change it to `PingInterval time.Duration`?

Reply:
changed.